### ISSUE TITLE
Auth: Password Expiration Check - Corrections to English language file

### ISSUE THREAD

**Issue by michael-milette** [DBezemer/moodle-auth_pwdexp#1]
Here are a couple of strings from the en/auth_pwdexp.php file that need to be updated. 

CHANGE #1

The first change improves the clarity of the text and corrects a typo in the word expire (not exipire).

Is currently:

```
$string['auth_pwdexpdescription'] = "This authenticator checks if the password of the user needs to exipire.<br/>If so, it will set the flag to force the account to change its password and redirect to the given URL.<br/><br/>Be sure to save these settings at least once and after each change.";
```

Should be:

```
$string['auth_pwdexpdescription'] = "This authenticator checks if the user's password needs to expire.<br/>If so it will set the flag to force the account to change it's password and redirect to the given URL.<br/><br/>Be sure to save these settings at least once and after each change.";
```

CHANGE #2

This change is a re-write of a string that was missing a space.

Is currently:

```
 $string['auth_expirationdays_key'] = "Expirationdays";
```

Should be:

```
 $string['auth_expirationdays_key'] = "Days until expiry";
```

I would appreciate it very much if you could integrate these 2 changes into future releases of auth_pwdexp.

Feel free to contact me if you have any questions or concerns.

Best regards,

Michael Milette

**Comment by DBezemer** [comment]
Michael,

can you create a pull request for these? I will merge them automatically then.
If not I'll merge the changes manually.

David

**Comment by michael-milette** [comment]
Thanks David.

**Comment by DBezemer** [comment]
merged, and closing as it's fixed in #2
